### ISSUE TITLE
setupAcceptanceTest callbacks is second parameter

### ISSUE THREAD

**Issue by ming-codes** [emberjs/ember-mocha#131]
The doc in [README](https://github.com/emberjs/ember-mocha/blob/master/README.md#acceptance-tests) shows `setupAcceptanceTest` accepts a single parameter, but it looks like `setupAcceptanceTest` is just an alias for `setupTest` where the callbacks is the second parameter.

**Comment by Turbo87** [comment]
@ming-codes sorry, I'm not sure what you're saying. is something not working as expected?

**Comment by ming-codes** [comment]
I guess another another way of saying it is..doing what the README describes throws error.

```js
import Ember from 'ember';
import { describe, it } from 'mocha';
import { setupAcceptanceTest } from 'ember-mocha';

var Application = Ember.Application.extend({
  rootElement: '#ember-testing',
});

describe('basic acceptance test', function() {
  setupAcceptanceTest({ Application });

  it('can visit /', function() {
    visit('/');

    return andThen(() => {
      expect(currentURL()).to.equal('/');
    });
  });
});
```

`setupAcceptanceTest` actually expects a first argument of type string.

```js
// This works.
setupAcceptanceTest('', { Application });
```

**Comment by Turbo87** [comment]
hmm, I thought I fixed that

**Comment by ming-codes** [comment]
I'm on `0.11.0` and this is still happening.

**Comment by Turbo87** [comment]
hmm, we are actually testing for this case (see https://github.com/emberjs/ember-mocha/blob/master/tests/setup-acceptance-test-test.js#L26) and in fact https://github.com/emberjs/ember-mocha/commit/2368d75be7375cbc0b24aab22a620eb3c2d1f687 fixed exactly the case that you're presenting... 🤔

**Comment by ming-codes** [comment]
That is not how my setupTest looks like 😦 

```js
define('ember-mocha/setup-test-factory', ['exports', 'mocha', 'ember-test-helpers'], function (exports, _mocha, _emberTestHelpers) {
  'use strict';

  exports['default'] = function (Constructor) {
    return function setupTest(moduleName, options) {
      var module;

      (0, _mocha.before)(function () {
        module = new Constructor(moduleName, options);
      });

      (0, _mocha.beforeEach)(function () {
        var _this = this;

        return module.setup().then(function () {
          var context = (0, _emberTestHelpers.getContext)();
          Object.keys(context).forEach(function (key) {
            _this[key] = context[key];
          });
        });
      });

      (0, _mocha.afterEach)(function () {
        return module.teardown();
      });
    };
  };
});
```

Can you check if that change got released?

**Comment by Turbo87** [comment]
looks like you're right. this simply hasn't been released yet... :shipit:

**Comment by Turbo87** [comment]
this is now released as `v0.12.0`
